### PR TITLE
Some more modifications, some issues fixed

### PR DESCRIPTION
**Pull request by MikeSchumacherCapgemini**
Modified code to execute code from functions.ftl instead of from OaspUtil to generate getters/setters

Fixed issues: Creation of fields+getters/setters with incorrect number of methods

### CODE REVIEW

File: cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <#ftl ns_prefixes={"xmi":"http://schema.omg.org/spec/XMI/2.1"}>
 <#include '/functions.ftl'>
 <#assign name = elemDoc["self::node()/@name"]>
-<#assign connectors = doc["xmi:XMI/xmi:Extension/connectors/connector"]>
```
Comment (jdiazgon) [inline cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl:4]: Have you tested this? I see that you removed the variable `connectors` but then you use it on line 14:
`<#list connectors as connector>`

**Review by jdiazgon — CHANGES_REQUESTED** [review]

File: cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl (hunk 1 above)
Comment (MikeSchumacherCapgemini) [reply cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl:4]: I moved the assignment to functions.ftl so you don't have to assign it every time you want to use them

File: cobigen-templates/templates-oasp4j/src/main/java/com/devonfw/cobigen/templates/oasp4j/utils/OaspUtil.java [hunk 2]
```diff
@@ -684,4 +684,8 @@ public String getRelationShipAnnotation(Map<String, Object> rs, String entityNam
             return "";
         }
     }
+
+    public void print(String s) {
+        System.out.println(s);
```
Comment (jdiazgon) [inline cobigen-templates/templates-oasp4j/src/main/java/com/devonfw/cobigen/templates/oasp4j/utils/OaspUtil.java]: What is this method for? Is it for testing?

File: cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl [hunk 3]
```diff
@@ -30,7 +29,7 @@ public class ${variables.className}Eto extends <#if variables.className?contains
 
   <#-- Generates all the connnected classes -->
   <#-- For generating the variables and methods of all the connected classes to this class -->
-    ${OaspUtil.generateConnectorsVariablesMethodsText()}
+    <#-- ${OaspUtil.generateConnectorsVariablesMethodsText()} -->
```
Comment (jdiazgon) [inline cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl:32]: Why do you remove this? 

Was it not working?

**Comment by jdiazgon** [comment]
> Modified code to execute code from functions.ftl instead of from OaspUtil

Actually @MikeSchumacherCapgemini , we should use more Java code rather than functions.ftl. One of our objectives is to stop using functions.ftl because it's harder to understand and is less flexible.

File: cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl (hunk 1 above)
Comment (jdiazgon) [reply cobigen-templates/templates-oasp4j/xml_ea/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.className}Eto.java.ftl:4]: Good idea 👍

**Review by jdiazgon — APPROVED** [review]

**Pull request merged**